src/components/calcite-popover/calcite-popover.e2e.ts

```diff
@@ -71,7 +75,7 @@ describe("calcite-popover", () => {
     const page = await newE2EPage();
 
     await page.setContent(
-      `<calcite-popover placement="auto"></calcite-popover><div>referenceElement</div>`
+      `<style>.hydrated--invisible {visibility: hidden;}</style><calcite-popover placement="auto"></calcite-popover><div>referenceElement</div>`
```

Comment:
Does this not get pulled in from the includes?

Reply:
No it doesn't seem to get pulled in. At least not for the tests. Do you want to dig more?

Reply:
> Do you want to dig more?

Maybe later, for now I'll leave the shovel in your hands. 😄 I was curious as I'm under the impression that we could depend on CSS being applied as some E2E tests may depend on styling.

Reply:
Created ticket for Stencil.